Controllers/ProductController.js

```diff
@@ -6,14 +6,15 @@ const DataCheckers = require("../Services/SanitiseValidateService");
 const getAllProducts = async(req, res) => {
     const connection = await DbService()
     const products = await ProductService.getAllProducts(connection)
-
-    const apiResponse = products.length > 0
-        ? JsonResponseService(products, true, 'Success', 200)
-        : JsonResponseService()
-
+    let apiResponse
+    products.length > 0
+        ? apiResponse = JsonResponseService(products, true, 'Success', 200)
+        : apiResponse = JsonResponseService()
     res.json(apiResponse)
 }
 
+
```

Comment:
I think we should delete 3xline spaces here

Reply:
NP